src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -527,6 +534,25 @@ public function toggleStatusAction($customerId)
         return $this->redirectToRoute('admin_customers_index');
     }
 
+    /**
+     * @AdminSecurity(
+     *     "is_granted('update', request.get('_legacy_controller'))",
+     *     redirectRoute="admin_customers_index",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param int $customerId
+     *
+     * @return JsonResponse
+     */
+    public function toggleStatusThenReturnJsonAction($customerId)
+    {
+        return $this->performActionThenReturnJsonResponse(
+            'toggleCustomerStatus',
+            (int) $customerId
```

Comment:
```suggestion
            $customerId
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/25115/commits/5aabb5bb639e79844a62601873be667ab4bb33ab